diagnostic.md

```diff
@@ -150,7 +187,9 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+All divs will float left, lining them all up in a row: divOne thru divFive.
+divOne then goes to the top right corner.
+divFour then goes to the bottom left corner.
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short: 2 3 1 4 5